speech/tests/system.py

```diff
@@ -245,3 +247,41 @@ def test_stream_recognize_single_utterance(self):
             for results in self._make_streaming_request(
                     file_obj, single_utterance=False):
                 self._check_results(results.alternatives)
+
+    def test_long_running_recognize(self):
+
+        content = None
+        reps = 5
+        with open(AUDIO_FILE, 'rb') as in_stream:
+            in_wavfile = wave.open(in_stream)
+            params = in_wavfile.getparams()
+            frames = in_wavfile.readframes(in_wavfile.getnframes())
+
+        with io.BytesIO() as out_stream:
+            out_wavfile = wave.open(out_stream, 'w')
+            out_wavfile.setparams(params)
+            for rep in range(reps):
+                out_wavfile.writeframes(frames)
+                content = out_stream.getvalue()
+
+        client = speech.SpeechClient()
+
+        operation = client.long_running_recognize(
+            config=speech.types.RecognitionConfig(
+                encoding='LINEAR16',
+                language_code='en-US',
+            ),
+            audio=speech.types.RecognitionAudio(content=content)
+        )
+
+        try:
+            op_result = operation.result()
+        except GaxError:
+            self.fail()
+        self.assertGreater(len(op_result.results), 0)
+        for result in op_result.results:
+            self.assertGreater(len(result.alternatives), 0)
+            for alternative in result.alternatives:
+                self.assertEqual(alternative.transcript.lower(),
+                                 ("hello thank you for using google cloud "
+                                  "platform " *reps)[:-1])
```

Comment:
Where does this "hello thank you..." string come from?

Reply:
moved it to its own variable describing where the string came from.